Portfolio/src/assets/WorkView.scss

```diff
@@ -0,0 +1,135 @@
+@import 'Variables.scss';
+
+.work-page-wrapper {
+  color: $color-black;
+  display: flex;
+  flex-direction: column;
+
+  .work-page-text-container {
+    /* font-family: "TitilliumWeb"; */
+    display: flex;
+    flex-direction: column;
+    margin: 7rem 2rem;
+
+    .work-page-title {
+      text-transform: uppercase;
+      font-weight: 600;
+
+      @media (min-width: 1900px) {
+        font-size: 30px;
+      }
+    }
+
+    .work-page-desc {
+      margin: 0;
+      line-height: 1.7rem;
+
+      @media (min-width: 768px) and (max-width: 1200px) {
+        width: $base-width * 30;
+      }
+
+      @media (min-width: 1900px) {
+        font-size: $font-size-p-desktop;
+      }
+    }
+
+    @media (min-width: 768px) and (max-width: 1200px) {
+      align-items: center;
+      text-align: center;
+      margin: 0;
+    }
+
+    @media (min-width: 1201px) and (max-width: 1899px) {
+      justify-content: flex-end;
+      font-size: $font-size-p-desktop;
+      width: $base-width * 30;
+      margin: 0;
+    }
+
+    @media (min-width: 1900px) {
+      justify-content: flex-end;
+      width: $base-width * 40;
+      margin-bottom: 5rem;
+    }
+  }
+
+  .work-page-projects-container {
+    display: flex;
+
+    .work-page-projects {
+      display: flex;
+      flex-direction: column;
+      gap: $gap-sm;
+      list-style: none;
+      text-transform: uppercase;
+
+      .work-page-project {
+        display: flex;
+        flex-direction: column;
+        gap: $gap-xs;
+
+        .project-label {
+          font-family: 'Raleway';
+          font-weight: 800;
+          letter-spacing: $letter-spacing-xs;
+          font-size: $font-size-span-mobile-md;
+
+          @media (min-width: 1201px) and (max-width: 1899px) {
+            font-size: $font-size-span-desktop;
+          }
+
+          @media (min-width: 1900px) {
+            font-size: $font-size-span-desktop-md;
+          }
+        }
+
+        .project-text {
+          @media (min-width: 1201px) and (max-width: 1899px) {
+            font-size: $font-size-span-mobile;
```

Comment:
You can see how it is a bit wrong, that u are using a font-size mobile for a desktop ratio

Reply:
Yeah, fixed it